### ISSUE TITLE
Error opening images.

### ISSUE THREAD

**Issue by IvyMasterDiego** [blueswen/mkdocs-glightbox#13]
When I click on the images, they don't load me correctly, I thought it was where the link was being obtained from but when I open those links, they don't load correctly, could you help me or something, here I put a screenshot what this looks like:

![imagen](https://user-images.githubusercontent.com/45220823/214056346-72dd5b9f-4f37-46cc-b6c9-2eb1b036125c.png)

**Comment by blueswen** [comment]
Could you provide your markdown code?

**Comment by blueswen** [comment]
How about a pseudo code? If all image on your docs does not load correctly, it may be related to the CSS in the mkdocs. Do you use another custom CSS to modify the image tag?

**Comment by IvyMasterDiego** [comment]
yes, this is the md: 
```
# H1

image 

![https://df7vkoxzcj1vg.cloudfront.net/4157d490-56f8-11ed-91e3-000c297ff65d/6f98129d-9646-4f78-9cea-dfccff4907e4]
```
and here is the css: 
```

  [data-md-color-scheme="slate"] {
    --md-primary-fg-color:        #0056b3;
    --md-primary-fg-color--light: #bbecb7;
    --md-primary-fg-color--dark:  #290390;
    --md-hue: 250; 
  }
```

**Comment by blueswen** [comment]
@IvyMasterDiego Sorry for the late reply. The bug has been fixed in the new version 0.3.2.
